pyproject.toml

```diff
@@ -1,28 +1,81 @@
-[tool.poetry]
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+
+[project]
 name = "docker-snapshot"
-homepage = "https://github.com/occamz/ds"
-documentation = "https://github.com/occamz/ds"
-version = "1.0.11"
+dynamic = ["version"]
 description = "`ds` is a development utility for managing snapshots inside a docker container."
-authors = ["occamz"]
 readme = "README.md"
+requires-python = ">=3.9"
+license = "Apache-2.0"
```

Comment:
Oh right. I absent-mindedly committed this with my defaults, but just noticed that there isn't a license to begin with. Thoughts on this, or should I replace this with `"Other/Proprietary License"` for now?

Reply:
Apache 2 is fine, I am not too well versed at OSS licences at this point, but seems sane from reading up on it for 5 minutes :D

Reply:
Cool, should I add the license file while I'm at it?

Reply:
Go for it 👍

Reply:
Done ✅